lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -167,7 +167,29 @@ def self.provision(hypervisor, options)
         beaker = Beaker::CLI.new.parse_options
         beaker.provision
         beaker.preserve_hosts_file
+        save_object(beaker.network_manager.hosts, "#{CONFIG_DIR}/hosts.yaml")
+        save_object(beaker.network_manager.hypervisors, "#{CONFIG_DIR}/hypervisors.yaml")
       end
+
+
+      def self.cleanup(hypervisor)
+        cli = Beaker::CLI.new.parse_options
+        cli.options[:hosts_file] = "#{CONFIG_DIR}/acceptance/config/default_#{hypervisor}_hosts.yaml"
+        network_manager = cli.create_network_manager
+        network_manager.hosts = load_object("#{CONFIG_DIR}/hosts.yaml")
+        network_manager.hypervisors = load_object("#{CONFIG_DIR}/hypervisors.yaml")
```

Comment:
hmmm...I'm not sure what this `hypervisors.yaml` is, or this`hosts.yaml`...oh, but I now get why the network manager is made readable. Given the changes from #1357 I would think that you would just be able to pull in `.beaker/subcommand_options.yaml`, which should happen during parsing...